mtags/src/main/scala-3/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -33,6 +34,12 @@ object AutoImports:
      */
     case Renamed(sym: Symbol, ownerRename: String)
 
+    /**
+     * Rename symbol itself, import only.
+     * `Map -> ("", import java.{util => ju})`
```

Comment:
```suggestion
     * `Map -> ("ju.Map", no-import)`
```

Reply:
No, `SelfRenamed` is for `import` only, and it won't complete anything (like `ju.Map`)